iib/workers/tasks/build.py

```diff
@@ -702,17 +706,26 @@ def _prepare_request_for_build(
             ocp_version = (
                 get_image_label(from_index_resolved, 'com.redhat.index.delivery.version') or 'v4.5'
             )
+            resolved_distribution_scope = (
+                get_image_label(from_index_resolved, 'com.redhat.index.delivery.distribution_scope')
+                or 'prod'
```

Comment:
You're storing this as upper-case in the database but the default is lower-case here. I realize that `_validate_distribution_scope`  sets it to upper-case again, but this just seems weird and unnecessary.